src/NzbDrone.Core/Download/Clients/qBittorrent/QBittorrent.cs

```diff
@@ -0,0 +1,252 @@
+                    case "error": // some error occurred, applies to paused torrents
+                        item.Status = DownloadItemStatus.Failed;
+                        item.Message = "QBittorrent is reporting an error";
+                        break;
+
+                    case "pausedDL": // torrent is paused and has NOT finished downloading
+                        item.Status = DownloadItemStatus.Paused;
+                        break;
+
+                    case "queuedDL": // queuing is enabled and torrent is queued for download
+                    case "checkingDL": // same as checkingUP, but torrent has NOT finished downloading
+                        item.Status = DownloadItemStatus.Queued;
+                        break;
+
+                    case "pausedUP": // torrent is paused and has finished downloading
+                    case "uploading": // torrent is being seeded and data is being transfered
+                    case "stalledUP": // torrent is being seeded, but no connection were made
+                    case "queuedUP": // queuing is enabled and torrent is queued for upload
+                    case "checkingUP": // torrent has finished downloading and is being checked
+                        item.Status = DownloadItemStatus.Completed;
+                        item.RemainingTime = TimeSpan.Zero; // qBittorrent sends eta=8640000 for completed torrents
+                        break;
+
+                    case "stalledDL": // torrent is being downloaded, but no connection were made
+                        item.Status = DownloadItemStatus.Warning;
+                        item.Message = "The download is stalled with no connections";
+                        break;
+
+                    case "downloading": // torrent is being downloaded and data is being transfered
+                    default: // new status in API? default to downloading
+                        item.Status = DownloadItemStatus.Downloading;
+                        break;
+                }
+
+                queueItems.Add(item);
+            }
+
+            return queueItems;
+        }
+
+        public override void RemoveItem(string downloadId, bool deleteData)
+        {
+            _proxy.RemoveTorrent(downloadId, deleteData, Settings);
+        }
+
+        public override DownloadClientStatus GetStatus()
+        {
+            var config = _proxy.GetConfig(Settings);
+
+            var destDir = new OsPath((string)config.GetValueOrDefault("save_path"));
+
+            return new DownloadClientStatus
+            {
+                IsLocalhost = Settings.Host == "127.0.0.1" || Settings.Host == "localhost",
+                OutputRootFolders = new List<OsPath> { _remotePathMappingService.RemapRemoteToLocal(Settings.Host, destDir) }
+            };
+        }
+
+        private OsPath GetOutputPath(QBittorrentTorrent torrent)
+        {
+            // is save_path supported by /query/torrents?
+            if (torrent.SavePath != null)
```

Comment:
Is this the new property from your PR? Is there a new minimum API version that we can enforce and forego the block below? I'm quite happy forcing a minimum version for a new release.